packages/office-ui-fabric-react/src/components/Button/sass/ButtonBase.scss

```diff
@@ -46,11 +46,4 @@
   .ms-Button-description, .ms-Button-icon {
```

Comment:
I'm  happy to remove them, but some of our example buttons are passing  in descriptions expecting  them  to be hidden by this css. I'd prefer each button having it's own props, and restrict which ones have descriptions.